### PR TITLE
Wait for the moderator if user have no permissions to create a whiteboard

### PR DESCRIPTION
**Pull request by maheichyk**
User may not have all the permissions to create a whiteboard. Currently he needs permissions to send `m.room.power_levels` and `net.nordeck.whiteboard`.

In this case he will have to wait for the moderator to join to proceed.

wait_moderator

**:heavy_check_mark: Checklist**

- [x] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [x] Added or updated documentation.
- [x] Tests for new functionality and regression tests for bug fixes.
- [x] Screenshots or videos attached (for UI changes).
- [x] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).

### CODE REVIEW

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Tested and works. Code looks good :+1: 

My main question is where the documentation should go to.

File: src/components/common/PageLoader.tsx [hunk 1]
```diff
@@ -14,9 +14,16 @@
  * limitations under the License.
  */
 
-import { Box, CircularProgress } from '@mui/material';
+import { Box, CircularProgress, Typography } from '@mui/material';
+import { unstable_useId as useId } from '@mui/utils';
```
[the remark this answers is not in the document]
Comment (weeman1337) [reply src/components/common/PageLoader.tsx:18]: Only available from React 18

File: docs/configuration.md [hunk 2]
```diff
@@ -21,3 +21,12 @@ REACT_APP_MULTISELECT=false
 # Does only work in development mode.
 REACT_APP_DEVTOOLS=true
 ```
+
+## Room power level
```
[the remark this answers is not in the document]
Comment (weeman1337) [reply docs/configuration.md:25]: Room configuration may also be considered as configuration

**Review by weeman1337 — APPROVED** [review]

File: docs/configuration.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (maheichyk) [reply docs/configuration.md:25]: okay, did some small changes to documentation to make it a bit more clear

**Pull request merged**